### PR TITLE
[k12-kit] Create EDA net new org flow

### PR DESCRIPTION
**Pull request by jofsky**
…'s net_new_org flow for Dina

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: cumulusci.yml [hunk 1]
```diff
@@ -235,6 +235,27 @@ flows:
...
+                task: install_managed_beta
+            3:
+                flow: config_managed
+            4:
+                task: enable_course_connections
+            5:
+                task: deploy_trial_config
+                options:
+                    unmanaged: False
+            6:
```
Comment (jstvz) [inline cumulusci.yml]: Duplicate. `update_admin_profile` is included in `config_managed`.

**Comment by dgamboni** [comment]
@jstvz @jofsky I'm thinking we might want to use the `install_beta` flow here, which would include steps 1-3 already listed and also add the `snapshot_changes` task.

1) flow: install_beta [from current folder]
    1) flow: dependencies
        1) task: update_dependencies
        2) task: deploy_pre
    2) task: install_managed_beta
    3) flow: config_managed
         1) task: deploy_post
         2) task: update_admin_profile
         3) task: deploy_k12_kit_settings
    4) task: snapshot_changes
2) task: enable_course_connections
3) task: deploy_trial_config
4) task: add_application_type_picklist_values

Thoughts?

File: cumulusci.yml [hunk 2]
```diff
@@ -235,6 +235,25 @@ flows:
             6:
                 task: add_application_type_picklist_values
 
+    net_new_org:
+        description: Simulates an K12 installation for a new customer. Installs the latest beta and dependencies.
+        group: 'K-12: Trial'
```
[the remark this answers is not in the document]
Comment (jofsky) [reply cumulusci.yml:240]: None, removing now.

File: documentation/automation.md [hunk 3]
```diff
@@ -53,11 +55,12 @@ Each directory is used as follows:
 
 ## Utility Tasks and Flows
 
--   **`upgraded_org`** Simulates an org that has been push-upgraded to the latest releases of k12 and underlying dependencies.
+-   **`regression_org`** Simulates an K12 org upgraded from the latest production release to the current beta and its dependencies (EDA), using the unpackaged metadata from the current beta. Use this when you want an upgraded org without needing to make any manual configurations.
+
+-   **`net_new_org`** Simulates the creation of a new K12 org for a new customer, installing the latest beta of K12 and dependencies (EDA). Use this when you want a fully configured regression environment that matches a new installation.
```
Comment (jstvz) [inline documentation/automation.md:60]: May want to mention that this flow's output is more like the trial than the installer, but fine as is.

**Pull request merged**